clang/lib/CodeGen/TargetBuiltins/PPC.cpp

```diff
@@ -294,6 +294,34 @@ Value *CodeGenFunction::EmitPPCBuiltinExpr(unsigned BuiltinID,
     return Builder.CreateCall(F, Ops, "");
   }
 
+  // BCD convert builtins for P9
+  case PPC::BI__builtin_ppc_national2packed:
+  case PPC::BI__builtin_ppc_packed2zoned:
+  case PPC::BI__builtin_ppc_zoned2packed: {
+    SmallVector<Value *, 3> Ops;
+    // Zero extending unsigned char to 32 bits by using Int32ty
```

Comment:
Always opt to doc the intention of the code vs just doc what the code does.
```suggestion
    // Zero extend unsigned char to Int32ty to match BE intrinsic definition.
```